src/app/ui/doc_view.cpp

```diff
@@ -653,4 +662,9 @@ void DocView::onCancel(Context* ctx)
   }
 }
 
+bool DocView::hasContentInActiveFrame(const doc::Layer* layer) const
+{
+  return (layer && layer->cel(m_editor->frame()));
+}
```

Comment:
Here if the `layer` is a `Group`, this function returns `false`, but this layer group can contain some layer with a cel. Then`DocView::onAfterLayerVisibilityChange()` doesn't invalidate properly the editor.
Recommended change:
````cpp
bool DocView::hasContentInActiveFrame(doc::Layer* layer) const
{
  if (layer && !layer->isGroup())
    return (layer && layer->cel(m_editor->frame()));

  // Recursive browsing of layers if 'layer' is a Group of Layers
  auto layers = dynamic_cast<LayerGroup*>(layer)->layers();
  for (auto& l : layers) {
    if (hasContentInActiveFrame(l))
      return true;
  }
  return false;

}```